### PR TITLE
Bug 1115480 - Part 3: UI test of Presentation API; r=johnhu

### PR DESCRIPTION
**Pull request by xeonchen**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4517a0a1aa65e4e89a0c45896e6184a82de0bd76

File: dev_apps/uitest/js/API/presentation_api.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function() {
+  function log(aMsg) {
+    dump('-*- mdns.js : ' + aMsg + '\n');
```
Comment (huchengtw-moz) [inline dev_apps/uitest/js/API/presentation_api.js]: Please use `console.log` here.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5d82657af900b684cddc556aeb4f4ba3e729d625

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=77eaf377ed6b15b38d68c2f6f0766c005876e1d3

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=984a46ddeded7139559b84181bea8eb98c48117b

**Pull request merged**